data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you step off your ship a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as David and Goliath myself."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	“You fancy yourself a savior or a prophet then?” the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused incredulous smile.`
```

Comment:
```suggestion
			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
```

Or even explicitly "and" instead of the comma. Dealer's choice :)